tools/bin/ecflow/make_suite_def.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
+"""
+make_suite_def.py
+"""
+import os
+import os.path
+import sys
+import argparse
+
+import ecflow
+
+from tonic.io import read_config
+from monitor.log import set_logger
+
+# --------------------------------------------------------------------------- #
+def main():
+    """
+    Write suite definition file for over-the-loop data processing.
+    """
```

Comment:
1. this isn't overtheloop.
1. How is this file different than ` tools/bin/ecflow/def_files/make_suite_def.py `?